src/woo_publications/publications/api/serializers.py

```diff
@@ -344,6 +377,26 @@ def update(self, instance, validated_data):
 
         return publication
 
+    @transaction.atomic
+    def create(self, validated_data):
+        ic_queryset = InformationCategory.objects.filter(
+            id__in=[ic.pk for ic in validated_data["informatie_categorieen"]]
+        )
+        retention_ic = get_retention_informatie_category(ic_queryset)
+
+        if retention_ic:
```

Comment:
Could change to an assert because this should always have a record but this seemed saver?